packages/rsocket-core/src/RSocketResumableTransport.js

```diff
@@ -386,14 +391,6 @@ export default class RSocketResumableTransport implements DuplexConnection {
   _writeFrame(frame: Frame): void {
     // Ensure that SETUP frames contain the resume token
     if (frame.type === FRAME_TYPES.SETUP) {
-      invariant(
-        frame.majorVersion > 1 ||
-          (frame.majorVersion === 1 && frame.minorVersion > 0),
-        'RSocketResumableTransport: Unsupported protocol version %s.%s. ' +
-          'This class implements the v1.1 resumption protocol.',
-        frame.majorVersion,
-        frame.minorVersion,
-      );
```

Comment:
rsocket-java expects 1.0 - protocol was not released yet, and also because this class  is likely not widely used, given there is no server resumption support in js, and c++ tracks resumable frames as byte count instead of positions (a-la "1.0" protocol version, but this clarification was never merged).